.github/workflows/e2e.yml

```
@@ -0,0 +1,56 @@
+on: push
+
+jobs:
+  e2e:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
```

actionlint (1.7.11)

[error] property "node-version" is not defined in object type {}

(expression)